### PR TITLE
feat: home page

### PR DESCRIPTION
**Pull request by GastonGutierrezC**
US 1: Home page
Description : 
In this task we worked on the initial page of the application, where the window is displayed in both desktop and mobile versions, we added images and buttons or icons when you click on them, it prints what you see in the image or icon for the moment.

### CODE REVIEW

**Comment by Miguelito2774** [comment]
My observations:
- The resolution should be that of the Figma.
- You must respect atomic design: atoms, molecules, organisms, templates and page.
- The idea is not to have a footer, but a navbar that can be adapted to both desktop and mobile, a single component should serve us for both applying conditional rendering.
- You must apply semantic HTML.
- You must refactor HomePage, and divide it into reusable components, respecting atomic design, you should have the HomePage only with calls to components, not a bunch of divs as it is now, in short repeated code that can be componetized (I mentioned it to you in person), so your code is more readable.
- Each class you make should have a single responsibility (render the component in question), you should not have several components in a single file, maximum you can use auxiliary functions(is not the same as components).

**Comment by JheremyTancara** [comment]
My observations:
* You have to separate the images into separate folders.

**Review by JheremyTancara — APPROVED** [review]

File: apps/frontend/app/Header/NavBarDisplay.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+
+import Header from "./NavBar";
```
Comment (Miguelito2774) [inline apps/frontend/app/Header/NavBarDisplay.tsx]: Just refactor this component name.

File: apps/frontend/app/globals.css [hunk 2]
```diff
@@ -25,3 +25,15 @@ body {
...
+  filter: brightness(0) invert(1); 
+}
+
+.icon-active {
+  filter: brightness(0) saturate(100%) hue-rotate(-50deg) brightness(1.2);
+}
+
+.custom-row-span {
+  min-height: 100px; 
+}
```
Comment (Miguelito2774) [inline apps/frontend/app/globals.css]: Add end-line please

**Review by Miguelito2774 — APPROVED** [review]
Good!!

File: apps/frontend/app/Header/NavBarDisplay.tsx (hunk 1 above)
Comment (GastonGutierrezC) [reply apps/frontend/app/Header/NavBarDisplay.tsx]: Done

File: apps/frontend/app/globals.css (hunk 2 above)
Comment (GastonGutierrezC) [reply apps/frontend/app/globals.css]: Done

**Pull request merged**